scripts/install-test-dependencies.sh

```diff
@@ -1,6 +1,7 @@
 #!/bin/bash
 set -e
 
-pip3 install  wheel pyyaml requests --force-reinstall
-pip3 install  fetchai-ledger-api==0.11.0a3 --force-reinstall
+pip3 install wheel --force-reinstall
+pip3 install pyyaml requests --force-reinstall
```

Comment:
pyyaml needs wheel to install without errors, but pip is not clever enough to know that.